addons/binding/org.openhab.binding.autelis/pom.xml

```diff
@@ -1,18 +1,17 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just a quick check is this how the skeleton makes them nowadays?

Reply:
In the near future when https://github.com/eclipse/smarthome/pull/5380 is merged. :slightly_smiling_face: